README.md

```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
 $ git worktree add vsc
 $ git worktree add wip
 ```
+
+## Commits with worktrees
+
+Pushing/pulling changes in worktrees is no different than in a regular repo. As soon as you change directory into a worktree folder, you can work as if you were on a regular repo. There will be an active branch, you can create/checkout other branches and commit to any branch as usual.
+
+### Unreviewed branches
+
+Branches such `wip` do not require PRs and reviews to push changes.
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/wip
+```
+2. Add and commit the files affected by this change
+```
+$ git add 00_test/TEMPLATE.eb
+$ git commit -m "adding WIP easiconfig TEMPLATE.eb"
+```
+3. Push new commit to remote branch in vsc-software-stack repo
+```
+$ git push origin wip
+```
+
+## Reviewed branches
+
+The `vsc` requires a PR and a positive review to merge changes into it.
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/vsc
+```
+2. Create a new local branch to work on the changes
+```
+$ git branch issue-00
+$ git checkout issue-00
```

Comment:
```suggestion
$ git checkout -b 000_example

Reply:
fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/f4ae9ff292204230adaaf531ba58643b6661ccb3